### ISSUE TITLE
Bad gateway error on login

### ISSUE THREAD

**Issue by zarak** [digitallyinduced/ihp-forum#6]
As a logged out user, I clicked on the new comment button and was taken to the log-in page. When I entered my login information, I received a 502 Bad Gateway error. When I navigated back to the website, however, I was successfully logged in.

**Comment by mpscholten** [comment]
Thanks for reporting 👍 

It seems like the request query params are missing [when the sessions controller is doing the redirect after login](https://github.com/digitallyinduced/ihp/blob/master/IHP/AuthSupport/Controller/Sessions.hs#L93).

The url is saved to the session here: https://github.com/digitallyinduced/ihp/blob/master/IHP/LoginSupport/Helper/Controller.hs#L101 It think there the call to `getRequestUrl` will not pass the query arguments. Will take a closer look in a few minutes.

**Comment by mpscholten** [comment]
Just pushed a fix to IHP. We just need to update the IHP version of ihp-forum once this is merged into master.

**Comment by mpscholten** [comment]
With the latest deployment we're now running the latest IHP version again, so this should be fixed :)
